.gitignore

```diff
@@ -32,6 +32,7 @@ tmp/
 /tmp
 /bootstrap
 /qx
+/ville
```

Comment:
not sure it should be in framework

Reply:
This should be reverted

Reply:
Sorry about that. I reverted the commit and made a third commit with just the "svg" addition to ImageLoader